src/app/components/UserMenu/index.tsx

```diff
@@ -42,54 +41,55 @@ export default function UserMenu() {
       console.error(e);
     }
   }
-
   return (
     <Menu as="div" className="relative">
       <Menu.Button className="flex items-center text-gray-700 dark:text-white hover:text-black dark:hover:text-white transition-colors duration-200">
         <MenuIcon className="h-6 w-6" />
       </Menu.Button>
       <Menu.List position="left">
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("discover");
-          }}
-        >
-          <GlobeIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("discover")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("publishers");
-          }}
-        >
-          <TransactionsIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("websites")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            navigate("/send");
-          }}
-        >
-          <SendIcon className="w-6 h-6 -ml-0.5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("actions.send")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            navigate("/receive");
-          }}
-        >
-          <ReceiveIcon className="w-6 h-6 -ml-0.5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("actions.receive")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("transactions");
-          }}
-        >
-          <TransactionsIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("transactions")}
-        </Menu.ItemButton>
-        <Menu.Divider />
+        <div className="block lg:hidden">
```

Comment:
You shouldn't need `block` here, divs are block by default. 

```suggestion
        <div className="lg:hidden">
```